rta/at_command.py

```diff
@@ -17,7 +17,13 @@
 from . import RtaMetadata
 
 
-metadata = RtaMetadata(uuid="961d7a1f-7bad-41d5-a3d9-8e8a2f59a824", platforms=["windows"], endpoint=[], siem=[], techniques=[])
+metadata = RtaMetadata(
+    uuid="961d7a1f-7bad-41d5-a3d9-8e8a2f59a824",
+    platforms=["windows"],
+    endpoint=[],
+    siem=[],
```

Comment:
Do we have any rules to reference for this?

Reply:
This is just a linting modification, audit/adjusts to mappins on existing RTAs will be done in https://github.com/elastic/ia-trade-team/issues/62